releasing/README.md

```diff
@@ -141,7 +141,7 @@ go mod edit -require=sigs.k8s.io/kustomize/kyaml@$versionKyaml plugin/builtin/pa
 
 Create the PR:
 ```
-createBranch pinToKyaml "Pin to kyaml $versionKyaml"
+createBranch pinToKyaml "Update kyaml to $versionKyaml"
```

Comment:
We have always been removing these from the release notes, but these are obviously key dependencies, and it could be helpful to show them explicitly so folks can cross-reference the release notes if needed.